templates/firefox-ios.xliff

```diff
@@ -2008,6 +2032,14 @@
         <source>Clear Private Data</source>
         <note>Title displayed in header of the setting panel.</note>
       </trans-unit>
+      <trans-unit id="Settings.ClearSelectedWebsiteData.ClearOneItem.Button" xml:space="preserve">
+        <source>Clear 1 Item</source>
```

Comment:
I believe we still don't handle plurals correctly. See comment below on how we can join these two strings